bin/_mkasset.bat

```diff
@@ -24,30 +26,31 @@ title %MINDBENDER_PROJECTPATH% / %MINDBENDER_ASSET%
 cls
 
 echo+
-echo %MINDBENDER_ASSET% -----------
+echo   %MINDBENDER_ASSET% -----------
 echo+
-
-:: List available tasks
-setlocal enabledelayedexpansion
-set FOLDERQUERRY=%MINDBENDER_ASSETPATH%
-if {%FOLDERQUERRY%\work\*}=={} (
-	for /f %%i in ('dir %FOLDERQUERRY%\work /b ') do (
-   		set x=%%i
-   		echo   !x!
-	)
-)
-endlocal
-
 echo+
 echo   1. Type application and task.
 echo   2. Press [Enter] to launch application.
 echo+
 echo   For example:
 echo+
 echo   $ maya animation
-echo   $ houdini sim
 echo   $ nuke comp
 echo+
+
+:: List available tasks
+setlocal enabledelayedexpansion
+set FOLDERQUERRY=%MINDBENDER_ASSETPATH%
```

Comment:
the name of this [commit](https://github.com/mindbender-studio/core/pull/60/commits/b4ea425e675e26b3fae971f88cbd1ccf7340988c) should be "cleanup typo" not "cleanup type"
I am the greatest at typing.